core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -175,22 +178,90 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
 
     private fun setCivRankingsTable() {
         val majorCivs = gameInfo.civilizations.filter { it.isMajorCiv() }
-        val civRankingsTable = Table().apply { defaults().pad(5f) }
+
+        contentsTable.clear()
+        
+        if (UncivGame.Current.settings.useDemographics) contentsTable.add(buildDemographicsTable(majorCivs))
+        else contentsTable.add(buildRankingsTable(majorCivs))
+    }
+
+    enum class RankLabels { Demographic, Rank, Value, Best, Average, Worst}
+    private fun buildDemographicsTable(majorCivs: List<CivilizationInfo>): Table {
+        val demographicsTable = Table().apply { defaults().pad(5f) }
+        val aliveMajorCivs = majorCivs.filter{ it.isAlive() }
+
+        for (rankLabel in RankLabels.values())   {
+            if (rankLabel == RankLabels.Demographic) {
+                val demoLabel = Table().apply { defaults().pad(5f) }
+                demoLabel.add(rankLabel.name.toLabel()).row()
+                demoLabel.addSeparator().fillX()
+                demographicsTable.add(demoLabel)
+                for (category in RankingType.values()) {
+                    val headers = Table().apply { defaults().pad(5f) }
+                    headers.add(category.name.replace('_', ' ').toLabel()).row()
+                    headers.addSeparator().fillX()
+                    demographicsTable.add(headers)
+                }
+            } else {
+                demographicsTable.row()
+                demographicsTable.add(rankLabel.name.toLabel())
+
+                for (category in RankingType.values()) {
```

Comment:
Another approach to reduce indentation would be to extract the entirety of this for loop to a different function, `getRankingColumn(rankingType): Table` or something, which computes the column for that ranking type and returns it, so you can add it to the inner table.

Reply:
Done, moved to new function buildDemographicsHeaders.